controllers/controller.js

```diff
@@ -16,18 +22,31 @@ exports.getReviews = (req, res, next) => {
     .catch(next);
 };
 
-exports.getReviewsById = (req,res,next) => {
-    return selectReviewsById(req.params.review_id)
+exports.getReviewsById = (req, res, next) => {
+  return selectReviewsById(req.params.review_id)
     .then((review) => {
-        res.send({ review })
+      res.send({ review });
     })
-    .catch(next)
-}
+    .catch(next);
+};
 
-exports.getCommentsByReviewId = (req,res,next) => {
-    return selectCommentsByReviewId(req.params.review_id)
+exports.getCommentsByReviewId = (req, res, next) => {
+  return selectCommentsByReviewId(req.params.review_id)
     .then((comments) => {
-        res.send( { comments })
+      res.send({ comments });
     })
-    .catch(next)
-}
+    .catch(next);
+};
+
+exports.postCommentsByReviewId = (req,res,next) => {
+  const body = req.body.body;
+  const username = req.body.username;
+  const review_id = req.params.review_id;
+  const date = Date.now()
```

Comment:
You don't actually need to do this - if you check the seed file, you'll see that both votes and created_at default when inserting a new comment
I would snip these out Catherine. and the console log